sky_alert/openweather_service.py

```diff
@@ -97,6 +97,31 @@ def get_moonrise_moonset_from_json(self, lat: str, lon: str) -> MoonData:
             "Moonrise/moonset/moon phase data from OpenWeather does not match expected format."
         )
 
+    def get_cloud_data(self, lat: str, lon: str) -> CloudData:
+        """Get the following 24 hours of cloud data (current hour inclusive)."""
+
+        cloud_data = []
+
+        self.update_most_recent_weather(lat=lat, lon=lon)
+
+        json_data = self.most_recent_weather[(lat, lon)]
+
+        # Check for key existence at different levels
+        if "hourly" in json_data and isinstance(json_data["hourly"], list):
+            current_data = json_data["hourly"]
+            for i in range(0, 24):
```

Comment:
nit, but we could declare something like `HOURS_IN_DAY=24` ~~on top of the file~~ in the `constants.py` file and then do 
```suggestion
            for i in range(HOURS_IN_DAY):
```